WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -569,9 +570,8 @@ public async Task LoadWalletAsync()
 			}
 			finally
 			{
-				IsBusy = false;
 				MainWindowViewModel.Instance.StatusBar.RemoveStatus(loadingStatusText);
```

Comment:
Move this to the last line of try block above? I don't like the feeling to have an exception here. To much logic in RemoveStatus.

Reply:
There can be no exceptions in `RemoveStatus`. But I don't understand what you are suggesting. Can you make a PR?

Reply:
If it cannot than is is fine. My suggestion was to move the following code:
`MainWindowViewModel.Instance.StatusBar.RemoveStatus(loadingStatusText);`
below line 568.

Reply:
You have to put the `RemoveStatus` in a `finally` that has been opened with a `try` and immediately followed by a `AddStatus`, so putting it into that catch block makes no sense.  

However I made the add-remove, tryadd-tryremove, so you'll feel safer.